### PR TITLE
Add test for non-joined users peeking into multiple rooms

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

File: tests/30rooms/60anonymousaccess.pl [hunk 1]
```diff
@@ -192,6 +194,58 @@
    );
 
    test(
+      "Real user can call /events on another world_readable room",
```
Comment (leonerd) [inline tests/30rooms/60anonymousaccess.pl]: This probably wants to be "$name" not "Real"

File: tests/30rooms/60anonymousaccess.pl [hunk 2]
```diff
@@ -192,6 +194,58 @@
...
+               }),
+
+               await_event_not_history_visibility_or_presence_for( $nonjoined_user, $room_id2, [] )
+               ->then( sub {
+                  my ( $event ) = @_;
+
+                  assert_json_keys( $event, qw( content ) );
+                  my $content = $event->{content};
+                  assert_json_keys( $content, qw( body ) );
+                  $content->{body} eq "mice" or die "Want content body to be mice";
```
Comment (leonerd) [inline tests/30rooms/60anonymousaccess.pl]: Use `assert_eq()` for nicer error messages

**Comment by erikjohnston** [comment]
@leonerd PTAL

**Comment by erikjohnston** [comment]
(IRL: I should move this to `90jira` and reference SYN-606)

**Comment by leonerd** [comment]
@erikjohnston - for expedience of getting this done, should I move it?

**Comment by erikjohnston** [comment]
@leonerd Oh, yes, sorry. This is on my todo list but argh :fire:

**Comment by erikjohnston** [comment]
(that is, feel free)

**Comment by leonerd** [comment]
Done. :)

**Pull request merged**